From 24ba4beb5b776f12a30f8093b680eac43935ea99 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Mon, 14 Jun 2021 14:52:30 +0300
Subject: [PATCH] [SW-0] greco: increase boot fit size limit

Need to increase boot fit size limit otherwise greco boot fit
dynamic load fails in size check.

Current greco boot fit size ~1.34MB, so increased the limit from 1MB
to 2MB (kept some buffer for future growth).

Change-Id: I4641a0f24dadc6345df1832482c5fd86f95f5e09
---

diff --git a/include/memory.h b/include/memory.h
index 8be3476..d34e784 100644
--- a/include/memory.h
+++ b/include/memory.h
@@ -14,7 +14,7 @@
 #define MEM_GP_BASE		(SDRAM_BASE + 0x40000)	/* SRAM_BASE + 256K */
 #define MEM_GP_SZ		0xC0000			/* 768K */
 #define MEM_LKD_PCI_BASE	(SDRAM_BASE + 0x200000)	/* SRAM_BASE + 2M */
-#define MEM_LKD_PCI_SZ		0x100000		/* 1M */
+#define MEM_LKD_PCI_SZ		0x200000		/* 2M */
 #define MEM_FWUPD_PCI_BASE	(SDRAM_BASE + 0x600000)	/* SRAM_BASE + 6M */
 #define MEM_FWUPD_PCI_SZ	0x200000		/* 2M */
 #define MEM_I2C_BASE		(SDRAM_BASE + 0x800000)	/* SRAM_BASE + 8M */
